NeuroID/src/main/java/com/neuroid/tracker/utils/NIDVersion.kt

```diff
@@ -1,23 +1,30 @@
 package com.neuroid.tracker.utils
 
 import com.neuroid.tracker.BuildConfig
-import com.neuroid.tracker.NeuroID
 
 object NIDVersion {
-    fun getSDKVersion(): String {
+    fun getSDKVersion(isRN: Boolean, isAdvancedDevice: Boolean,
+                      nidBuildConfigWrapper: NIDBuildConfigWrapper = NIDBuildConfigWrapper()
+    ): String {
         val rnText =
-            NeuroID.getInternalInstance()?.isRN?.let {
-                if (it) {
-                    "rn-"
-                } else {
-                    ""
-                }
+            if (isRN) {
+                "rn-"
+            } else {
+                ""
             }
 
-        return "5.android-" + "$rnText" + BuildConfig.VERSION_NAME
+        val advText =
+            if (isAdvancedDevice) {
+                "-adv"
+            } else {
+                ""
+            }
+
+        return "5.android-$rnText${nidBuildConfigWrapper.getBuildVersion()}$advText"
```

Comment:
Open to discussion, should we do 3.3.0-adv or -adv-3.3.0? RN is appended pre the version number, should we stay consistent with that or?

Reply:
5.android-rn-adv-3.3.0